logging/logging-loki-ocp.adoc

```diff
@@ -0,0 +1,132 @@
+:_content-type: ASSEMBLY
+:context: logging-loki
+[id="logging-loki-ocp_{context}"]
+= Setting up logging
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+An essential function of {product-title} is the collection and aggregation of logs from the environments and application pods it runs.
+
+To achieve this, the {logging-title} uses Vector and Loki. Vector is a high-performance observability data pipeline that allows users to configure log forwarders to direct logs to different locations. Loki is a horizontally scalable, highly available, multi-tenant log aggregation system that indexes metadata about logs with labels, similar to Prometheus labels. The log data is compressed and stored in chunks in object stores or locally on the filesystem. Logging is designed to work with the default configuration, which is optimized for small to medium-sized clusters.
+
+In {logging} documentation, *LokiStack* refers to the {logging} supported combination of Loki, and web proxy with {product-title} authentication integration. LokiStack's proxy uses {product-title} authentication to enforce multi-tenancy. *Loki* refers to the log store as either the individual component or an external store. You can query Loki link:https://grafana.com/docs/loki/latest/[using JSON paths or regular expressions].
```

Comment:
> You can query Loki link:https://grafana.com/docs/loki/latest/[using JSON paths or regular expressions].

To me it sounds like this sentence should instead refer to the "LogQL" query language used in Loki. Destructuring JSON and matchers using regular-expressions are a part of it, but you can't directly throw JSONPath or a regex at the query engine:

> You can query Loki link:https://grafana.com/docs/loki/latest/logql/[using a query language called LogQL].